test/integration/__snapshots__/userFlow.test.js.snap

```diff
@@ -28,6 +28,14 @@ Array [
 
 exports[`createUser should not create duplicate accounts: no errors 1`] = `undefined`;
 
+exports[`createUser should raise an error without email in header: no email 1`] = `
+Array [
+  [GraphQLError: You must provide a vaild email],
```

Comment:
"vaild" should be valid, if you fix the snapshot you'll fix the typo on the code too.